locale/en/blog/vulnerability/mar-2022-security-releases.md

```diff
@@ -0,0 +1,50 @@
+---
+date: 2022-03-14T12:00:00.000Z
+category: vulnerability
+title: OpenSSL security releases may require Node.js security releases
+slug: openssl-and-high-severity-fixes-mar-2022
+layout: blog-post.hbs
+author: Joe Sepi
+---
+
+### Summary
+
+The Node.js project may be releasing new versions across all of its supported
+release lines late next week to incorporate upstream patches from OpenSSL.
```

Comment:
I think we were planning for late this week? (It was "next week" last week when we were having the discussions.)

Reply:
I'll open a PR to fix that. Sorry about the auto-merge.